### DISCUSSION TITLE
deploy the trained model on FastAPI getting the features from a pandas dataframe

### DISCUSSION THREAD

**Discussion by marcello-calabrese** [pycaret/pycaret#3315] [General]
Hi, in fastapi i need to make predictions using a model saved as pickle file, the input data are in a pandas dataframe, then the list datatypes of the predictions need to be included in the dataframe. How can I do it? thanks

**Comment by ngupta23** [comment]
check out the `create_api` feature in the documentation.

https://pycaret.gitbook.io/docs/get-started/functions/deploy#create_api

**Comment by marcello-calabrese** [comment]
Thanks, but I am not sure how to make prediction on batch data (via dataframe) using fastapi, i would like the endpoint to make predictions on a pandas dataframe .

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
